.github/workflows/template-ledger-epoch-snapshots.yml

```
@@ -178,71 +101,44 @@ jobs:
               --topology /config/topology.json
 
       - uses: ./.github/actions/cache
+        if: ${{ inputs.network == '' || inputs.network == matrix.network.name }}
 
       - name: Bootstrap DBs
+        if: ${{ inputs.network == '' || inputs.network == matrix.network.name }}
         uses: ./.github/actions/bootstrap-db
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/bootstrap-db": line 6: unexpected key "type" for definition of input "epoch"

(action)